spec/firebase_app_distribution_action_spec.rb

```diff
@@ -221,6 +221,7 @@
     before(:each) do
       allow(File).to receive(:exist?).and_call_original
       allow(action).to receive(:fetch_auth_token).and_return('fake-auth-token')
```

Comment:
is this still needed?

Reply:
Oh yeah I can get rid of that good call. Done!